src/subapps/projects/components/LinkActivityForm.tsx

```diff
@@ -73,7 +74,15 @@ const LinkActivityForm: React.FC<{
             </Col>
             {/* TODO: manage multiple types */}
             <Col>
-              <p>{labelOf(resourceType)}</p>
+              <p className="link-activity-form__types">
+                {resourceType && Array.from(resourceType).length > 0 ? (
+                  <TypesIconList
```

Comment:
What happens when resourceType is null/undefined etc ? could that happen?

Reply:
We just show nothing if it is the case. We check it here so the app doesn't blow up with error that something is not defined when it tries to render the component but data is still loading.